File: leetcode-algorithms/058. Length of Last Word/solution.cpp
#include <iostream>
#include <string>

using namespace std;

int lengthOfLastWord(string s) {
	int i = s.size() - 1;
	while(s[i]==' ' && i>=0){
		i--;
	}

	int res = 0;
	while(i>=0 && s[i]!=' '){
		i--;
		res++;
	}
	return res;


}

int main()
{
	return 0;
}

File: leetcode-algorithms/058. Length of Last Word/solution.py
def lengthOfLastWord(s):
	"""
	:type s: str
	:rtype: int
	"""
	s = s.strip()
	i = len(s) - 1
	res = 0
	while i>=0 and s[i]!=' ':
		i -= 1
		res += 1
	return res

		



if __name__ == '__main__':
	s = 'hello world'
	res = lengthOfLastWord(s)
	print(res)

